cmd/soroban-rpc/internal/methods/get_transactions.go

```diff
@@ -135,7 +144,7 @@ LedgerLoop:
 				err = errors.Errorf("ledger close meta not found: %d", ledgerSeq)
 			}
 			return GetTransactionsResponse{}, &jrpc2.Error{
-				Code:    jrpc2.InvalidParams,
+				Code:    jrpc2.InternalError,
```

Comment:
why internal now? if it's not in the DB then their request is out of bounds

Reply:
same comment for all of the below error changes imo

Reply:
@Shaptic Yeah you are correct. I refactored this code again to use InvalidParams for "ledger close meta not found" or for any other db related error InternalError code.

However, for these 3 errors:
- https://github.com/stellar/soroban-rpc/pull/136/files#diff-5276db8001730f32b267cab637b316958b69aef6f8184343338855fdc118b700R158
- https://github.com/stellar/soroban-rpc/pull/136/files#diff-5276db8001730f32b267cab637b316958b69aef6f8184343338855fdc118b700R169
- https://github.com/stellar/soroban-rpc/pull/136/files#diff-5276db8001730f32b267cab637b316958b69aef6f8184343338855fdc118b700R169

InternalError would make more sense dont you think? Since they are generated due to some internal parsing issues?